src/main/java/de/muenchen/isi/infrastructure/entity/stammdaten/FoerdermixStamm.java

```diff
@@ -13,15 +13,8 @@
 import javax.persistence.Column;
 import javax.persistence.Embedded;
 import javax.persistence.Entity;
-import javax.persistence.Table;
-import javax.persistence.UniqueConstraint;
 
 @Entity
-@Table(
-        uniqueConstraints = {
-                @UniqueConstraint(columnNames = {"bezeichnung"})
```

Comment:
Weshalb wurde das Unique-Constraint entfernt?

Das Constraint alleinig auf Basis des Attributs Bezeichnung ist nicht korrekt.

Jedoch sollte das Constraint auf Basis der Bezeichnung und des Jahres der Bezeichnung gemacht werden. Also als kombinierter fachlicher Schlüssel. Ansonsten ist es möglich Fördermixe mit der selben Bezeichnung für das selbe Jahr anzulegen. Dies wird in den Stammdaten zu Chaos führen.

```java
@Table(
        uniqueConstraints = {
                @UniqueConstraint(columnNames = {"bezeichnung", "bezeichnungJahr"})
        }
)

```

Reply:
#12 Duplikat